File: backend-python/backend-saude/app/routes/registerRoutes.py
# app/routes/registerRoutes.py

from flask import request, jsonify
from app.controller.registerController import register_user, update_biometric, list_funcionarios_for_biometric

def register_routes(app):
    app.add_url_rule('/register', 'register', register_user, methods=['POST'])
    app.add_url_rule('/update-biometric', 'update_biometric', update_biometric, methods=['POST'])
    app.add_url_rule('/funcionarios-biometric', 'list_funcionarios_biometric', list_funcionarios_for_biometric, methods=['GET'])

File: backend-java/src/main/java/br/com/bio/biomtria/route/registerRoutes.java
package br.com.bio.biomtria.route;

import br.com.bio.biomtria.controller.registerController;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.bind.annotation.*;
import org.springframework.http.ResponseEntity;

@RestController
@Configuration
public class registerRoutes {
    @PostMapping("/register")
    public ResponseEntity<?> registerUserApiPontoRoute(@RequestBody java.util.Map<String, Object> data) {
        return registerController.registerUser(data);
    }

    @Autowired
    private registerController registerController;

    @PostMapping("/api/ponto/register")
    public ResponseEntity<?> registerUserRoute(@RequestBody java.util.Map<String, Object> data) {
        return registerController.registerUser(data);
    }

    @PostMapping("/update-biometric")
    public ResponseEntity<?> updateBiometricRoute(@RequestBody java.util.Map<String, Object> data) {
        return registerController.updateBiometric(data);
    }

    @GetMapping("/funcionarios-biometric")
    public ResponseEntity<?> listFuncionariosRoute() {
        return registerController.listFuncionariosForBiometric();
    }
}
